tests/Functional/Schema/MySqlSchemaManagerTest.php

```diff
@@ -46,8 +46,8 @@ public function testSwitchPrimaryKeyColumns() : void
         $primaryKey = $table->getPrimaryKeyColumns();
 
         self::assertCount(2, $primaryKey);
-        self::assertContains('bar_id', $primaryKey);
-        self::assertContains('foo_id', $primaryKey);
+        self::assertArrayHasKey('bar_id', $primaryKey);
+        self::assertArrayHasKey('foo_id', $primaryKey);
```

Comment:
Why this change?

Reply:
Because before the methods were returning an array of column names. Now it's returning a HashMap/array indexed by column name => Column